.gitignore

```diff
@@ -0,0 +1,39 @@
+### Haskell ###
+
+dist
+dist-*
+cabal-dev
+*.o
+*.hi
+*.chi
+*.chs.h
+*.dyn_o
+*.dyn_hi
+.hpc
+.hsenv
+.cabal-sandbox/
+cabal.sandbox.config
+*.prof
+*.aux
+*.hp
+*.eventlog
+.stack-work/
+cabal.project.local
+cabal.project.local~
+.HTF/
+.ghc.environment.*
+.ghci
+
+### Cardano ###
+
+test_keystore.*
+state-staging
+state-integration
+secret.key
+secret.key.lock
+wallet-db-acid/
```

Comment:
Maybe start with a clean .gitignore? Or keep this?

Reply:
I would start with a clean one and as someone will be modelling db he/she will add a line to .gitignore before commiting

Reply:
I'd suggest to start with a clean one taken from: https://www.gitignore.io/